mesh/mesh.go

```diff
@@ -302,7 +352,7 @@ func (m *Mesh) ExtractUniqueOrderedTransactions(l *types.Layer) (validBlockTxs,
 
 	// Get and return unique transactions
 	seenTxIds := map[types.TransactionId]struct{}{}
-	return m.getTxs(uniqueTxIds(validBlocks, seenTxIds), l), m.getTxs(uniqueTxIds(invalidBlocks, seenTxIds), l)
```

Comment:
See other comments for why I think this code was not adequately migrated elsewhere.

What the removed code returns is the TxIDs that are only in invalid blocks. I don't think this logic was replaced anywhere.